lib/logstash/outputs/scalyr.rb

```diff
@@ -257,6 +261,9 @@ def get_new_metrics
   #
   public
   def multi_receive(events)
+    # Just return and pretend we did something if running in noop mode
+    return events if @noop_mode
```

Comment:
Do we actually need to return anything? Besides that, LGTM.

Reply:
I'm not 100% sure but in the real case we do return a list of the events that went through, though in a different format. I remember returning successful events being a pattern for a reason though the reasons behind it are long gone from my brain now. Either way it doesn't seem to cause any issues.